n3fit/src/n3fit/scripts/n3fit_exec.py

```diff
@@ -133,7 +133,7 @@ def from_yaml(cls, o, *args, **kwargs):
 
         N3FIT_FIXED_CONFIG['actions_'].append(namespace + "performfit")
 
-        if file_content["fitting"].get("savepseudodata"):
+        if file_content["fitting"].get("savepseudodata", True) == True:
```

Comment:
Why the `==True`? Since it is a default I think it would be good to have it be true unless explicitly set to `False`.
```suggestion
        if file_content["fitting"].get("savepseudodata", True):
```

Reply:
Because of the difference between the key existing and being true.

Reply:
Yes. My point is that if it exist and it is not explicitly `False` then it should default to True.

Reply:
But it doesn't work that way further down the pipeline (I didn't look at the code, just checked by running it)

Reply:
Ah, then it should fail if it is not a boolean.

Reply:
Well, I don't disagree but we do barely any typechecks, so I don't think that's for this PR to address. There must be lots of these situations in this repo if you'd actively go looking.

Reply:
Yes, there will be. But given that we just found one of those situations let's fix it.

This is happening at the level of `n3fit_exec` so nothing has been computed yet. It is a great place to fix the problem.